ci/taos/checker-pr-audit-async.sh

```diff
@@ -322,16 +322,19 @@ if [[ $global_check_result == "success" ]]; then
 
     # If contributors want later, let's inform developers of CI test result to go to a review process as a final step before merging a PR
     echo "[DEBUG] All audit modules are passed - it is ready to review! :shipit:. Note that CI bot has two sub-bots such as TAOS/pr-audit-all and TAOS/pr-format-all."
+    exit 0
```

Comment:
We need to move the existing  Line 337-340 in front of the if statement if you want to append exit xxx.

Reply:
Updated.